docs/developerguide/Headerfiles.md

```diff
@@ -0,0 +1,116 @@
+# 3. Headers Files
+
+## a) Create a `Module.h` header file
+
+- The file `Module.h` defines the mandatory `MODULE_NAME`. Thunder provides a trace and warning reporting feature. To accurately identify the source of a warning, Thunder needs to know the human-readable name of the package (executable or library). This package name is defined by the `MODULE_NAME` and declared by the `MODULE_NAME_DECLARATION()`.
+
+- It also includes wpeframework headers "<plugins/plugins.h>".
+    ```C
```

Comment:
Pls remove this IOController. We don't need it since there is example for FooPlugin below.